cookbook_se/recipes/sidecar/sidecar.py

```diff
@@ -0,0 +1,70 @@
+import os
+import time
+
+from flytekit import task, workflow
+from flytekit.taskplugins.sidecar import Sidecar
+from k8s.io.api.core.v1 import generated_pb2
+from k8s.io.apimachinery.pkg.api.resource.generated_pb2 import Quantity
+
+_SHARED_DATA_PATH = "/data/message.txt"
+
+
+# A simple pod spec in which a shared volume is mounted in both the primary and secondary containers. The secondary
+# writes a file that the primary waits on before completing.
+def generate_pod_spec_for_task():
+    pod_spec = generated_pb2.PodSpec()
+
+    primary_container = generated_pb2.Container(name="primary")
```

Comment:
the image here is automatically injected right?, we should explain that.
Also maybe used the {{.images...}} directive?

Reply:
good point, done